### DISCUSSION TITLE
Method to update data inside of a deployed application?

### DISCUSSION THREAD

**Discussion by smkacur** [Kitware/trame#519] [Q&A] [answered]
Hi, I am using docker to deploy a visualization application using trame and paraview. The data that we are visualizing are generated daily. Is there a way that I can update the data that is shown inside of the application so that it includes data that has been processed after the app was already deployed? Or will it be necessary to redeploy the app to capture the new data files?

I am using the --data arg and a json file to read data into the app (similar to this [example](https://github.com/Kitware/trame/tree/master/examples/07_paraview/TimeAnimation)) and am mounting the data files into the container at the time of deployment.

**Comment by jourdain** [comment, marked as answer]
As long as the data update within the same location of the mounted directory, it should just work. Every time a user connect, the data that is on the disk will be read. So if it has been updated, you will see the new content.

**Reply by smkacur** [reply]
That is very cool, thank you!
